src/ZombieDriver.php

```diff
@@ -94,13 +94,14 @@ public function stop()
     public function reset()
     {
         $js = <<<JS
-browser.destroy();
-browser = null;
+browser.headers = {};
+browser.deleteCookies();
 stream.end();
 JS;
 
         $this->server->evalJS($js);
-        $this->nativeRefs = array();
```

Comment:
I believe no, because as for Selenium all previously opened windows aren't closed and therefore all previously found elements are still there.